.github/workflows/ci.yml

```diff
@@ -37,3 +37,18 @@ jobs:
         with:
           github-token: ${{ secrets.github_token }}
           parallel-finished: true
+
+  semantic-release:
+    needs:  build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run semantic-release
+        if: github.repository == 'node-casbin/sequelize-adapter' && github.event_name == 'push'
+        run: |
+          yarn install --no-lockfile
+          yarn run prepack
+          yarn run release
```

Comment:
`release` didn't exist. You should first install `semantic-release` or use `npx`.

Reply:
thanks for help me find issue